docs/pipeline-components-and-applications/enrichment-components/4-0-x-upgrade-guide/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "4.0.x upgrade guide"
+sidebar_position: 0
+---
+
+## Breaking changes
```

Comment:
Should we also mention that the experimental rabbitmq artifact is no longer published?

Reply:
Yeah that doesn't hurt to mention it